conftest.py

```diff
@@ -73,3 +75,68 @@ def pytest_collection_modifyitems(config, items):
 
         if 'requires_bamboo' in item.keywords:
             item.add_marker(skip_outside_bamboo_test)
+
+
+@pytest.fixture()
```

Comment:
These fixtures do not belong at such a high part of the repo file tree. They are only used by your new tests which all reside in 1 file. Please include them in the file where they are used. If in future, we need them available across different files, we can move them to a `conftest.py` of a more appropriate scope, for example `/home/danielk/AllenSDK/allensdk/test/brain_observatory/conftest.py`

Reply:
Good suggestion, I moved the fixtures to the testing file where they are used.